unit_tests/OrthoTree/helpers/aid_list.cpp

```diff
@@ -231,74 +232,84 @@ TEST(AidListTest, NumParticlesInOctantTest) {
+            << "Failed exclusive upper bound test for octant: " << test.octant;
+
+        EXPECT_EQ(aid_list.getUpperBoundIndexInclusive(test.octant), test.expected_upper_bound_incl)
+            << "Failed inclusive upper bound test for octant: " << test.octant;
+
+        EXPECT_EQ(aid_list.getNumParticlesInOctant(test.octant), test.expected_total_particles)
+            << "Failed particle count test for octant: " << test.octant;
+    };
+
+    std::vector<TestStruct> tests_to_run = {// root node
+                                            {.octant                    = 0b000,
+                                             .expected_lower_bound      = 0,
+                                             .expected_upper_bound_excl = 0,
+                                             .expected_upper_bound_incl = 0,
+                                             .expected_total_particles  = total_num_particles},
+
+                                            // top-left octant
+                                            {.octant                    = 0b001,
+                                             .expected_lower_bound      = 0,
+                                             .expected_upper_bound_excl = 100,
+                                             .expected_upper_bound_incl = 99,
+                                             .expected_total_particles  = n_particles_per_proc},
+
+                                            // top-middle octant
+                                            {.octant                    = 0b011,
+                                             .expected_lower_bound      = 100,
+                                             .expected_upper_bound_excl = 200,
+                                             .expected_upper_bound_incl = 199,
+                                             .expected_total_particles  = n_particles_per_proc},
+
+                                            // left-middle octant
+                                            {.octant                    = 0b101,
+                                             .expected_lower_bound      = 200,
+                                             .expected_upper_bound_excl = 300,
+                                             .expected_upper_bound_incl = 299,
+                                             .expected_total_particles  = n_particles_per_proc},
+
+                                            // middle-middle octant
+                                            {.octant                    = 0b111,
+                                             .expected_lower_bound      = 300,
+                                             .expected_upper_bound_excl = 400,
+                                             .expected_upper_bound_incl = 399,
+                                             .expected_total_particles  = n_particles_per_proc}};
+
+    // #### TEST HELPERS DONE ####
+
```

Comment:
i moved this outside of the `rank == 0` bounds below, as the other ranks now also need this data